lib/AST/ASTImporter.cpp

```diff
@@ -1578,9 +1626,14 @@ Decl *ASTNodeImporter::VisitEmptyDecl(EmptyDecl *D) {
   // Import the location of this declaration.
   SourceLocation Loc = Importer.Import(D->getLocation());
 
-  EmptyDecl *ToD = EmptyDecl::Create(Importer.getToContext(), DC, Loc);
+  bool NewlyCreated;
```

Comment:
Making the pointer an output parameter and the NewlyCreated returned by CreateDecl would make these snippets more compact. I wonder if compactness or having no output parameter is better in this case.